lib/oci_utils/impl/oci-image-migrate-import-main.py

```diff
@@ -0,0 +1,256 @@
+# oci-utils
+#
+# Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown
+# at http://oss.oracle.com/licenses/upl.
+
+"""
+Script to import an on-premise virtual server from an object storage to the
+custom images in the Oracle Cloud Infrastructure. An image which passed the
+checks and modifications of - and was uploaded by the oci-image-migrate
+utility should import fine.
+"""
+import argparse
+import json
+import logging.config
+import os
+import sys
+
+from oci_utils.migrate import console_msg, exit_with_msg, read_yn
+from oci_utils.migrate import migrate_tools
+from oci_utils.migrate import migrate_utils
+
+if sys.version_info.major < 3:
```

Comment:
Why having such test as only our wrapper will run this and the package will require python3 ?

Reply:
for those bypassing the bash wrapper

Reply:
I guess thee is none then , no?